integration_tests/suite/helpers/database.py

```diff
@@ -97,7 +97,7 @@ def wrapped_function(self, *args, **kwargs):
             with self.database.queries() as queries:
                 recording['uuid'] = queries.insert_recording(**recording)
             try:
-                return func(self, recording, *args, **kwargs)
+                return func(self, *args, recording, **kwargs)
```

Comment:
:laughing: I see this one too in my branch, I hope it doesn't give you too much headache to find it